nms/packages/magmalte/app/views/equipment/EnodebDetailConfig.js

```diff
@@ -27,6 +27,7 @@ import JsonEditor from '../../components/JsonEditor';
 import React from 'react';
 import SettingsIcon from '@material-ui/icons/Settings';
 import nullthrows from '@fbcnms/util/nullthrows';
+import {NeighborCellConfig, NeighborFreqConfig} from './EnodebDetailNeighbor';
```

Comment:
Hi @hakeen0618 , I am trying out this new feature. When I try to build the NMS docker image, it is not able to find the EnodebDetailNeighbor.js file. Is it missing or am I doing somethin wrong? Thanks!

Reply:
Yes, I will update the PR right now.